### PR TITLE
INTLY-4011 - AMQ patch upgrade task

### PR DESCRIPTION
**Pull request by tremes**
## Verification Steps
As the verifier of the PR the following process should be done:

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

## Is an upgrade task required and are there additional steps needed to test this?

- [ ] Yes
- [ ] No

- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

**Comment by tremes** [comment]
@mikenairn @pmccarthy can you please check?

File: roles/enmasse/tasks/upgrade_images.yml [hunk 1]
```diff
@@ -1,4 +1,25 @@
----
-#ToDo Implement CVE image update steps as described https://github.com/RHCloudServices/integreatly-help/blob/master/sops/cves/applying-cve-updates.md
-- debug:
-    msg: "TODO Implement me!!"
+-
```
Comment (pmccarthy) [inline roles/enmasse/tasks/upgrade_images.yml]: Seems to be some indentation issues with this file? 

For example:
```
-
  name: "get all deployments"
```

Should this not be:
```
- name: "get all deployments"
```
?

Comment (tremes) [reply roles/enmasse/tasks/upgrade_images.yml]: Oh yeah you're probably right. I'll check tomorrow.Thx

**Pull request merged**